src/core/border_router/routing_manager.hpp

```diff
@@ -291,10 +291,13 @@ class RoutingManager : public InstanceLocator
     static constexpr uint32_t kMaxRtrAdvInterval           = 600;  // Max Router Advertisement Interval. In sec.
     static constexpr uint32_t kMinRtrAdvInterval           = kMaxRtrAdvInterval / 3; // Min RA Interval. In sec.
     static constexpr uint32_t kMaxInitRtrAdvInterval       = 16;                     // Max Initial RA Interval. In sec.
-    static constexpr uint32_t kRaReplyJitter               = 500;    // Jitter for sending RA after rx RS. In msec.
-    static constexpr uint32_t kRtrSolicitationInterval     = 4;      // Interval between RSs. In sec.
-    static constexpr uint32_t kMaxRtrSolicitationDelay     = 1;      // Max delay for initial solicitation. In sec.
-    static constexpr uint32_t kRoutingPolicyEvaluationJitter = 1000; // Jitter for routing policy evaluation. In msec.
+    static constexpr uint32_t kRaReplyJitter               = 500; // Jitter for sending RA after rx RS. In msec.
+    static constexpr uint32_t kRtrSolicitationInterval     = 4;   // Interval between RSs. In sec.
+    static constexpr uint32_t kMaxRtrSolicitationDelay     = 1;   // Max delay for initial solicitation. In sec.
+    static constexpr uint32_t kRoutingPolicyEvaluationJitterMin =
+        2000; // Min jitter for routing policy evaluation. In msec.
+    static constexpr uint32_t kRoutingPolicyEvaluationJitterMax =
+        4000; // Max jitter for routing policy evaluation. In msec.
```

Comment:
@simonlingoogle this looks fine to me, just curios what's the background/reason for this?

Reply:
This PR help to avoid the case, where a Thread BR would publish ULA prefix only to remove it immediately when the WiFi AP is changing the GUA prefix, e.g. because of a new GUA prefix being allocated from IA_PD.

Reply:
Thanks. Sounds good.

Reply:
@simonlingoogle please add the explanation to commit message :)